index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- NAVBAR HERE -->
+    <nav>
+      <ul>
+        <li class="logo-left">
+          <img src="img/03.png" alt="" srcset="">
+          <p>Original Trombones</p>
+        </li>
+        <li>Feature</li>
+        <li>How it works</li>
+        <li>Pricing</li>
+      </ul>
+    </nav>
+
+    <!-- Container Web Page -->
+
+    <div class="webapp-container">
+        <!-- EMAIL FORM -->
+
+      <div class="email-form">
+        <h2>Handcrafted, home made masterpieces</h2>
+        <input type="email" name="" placeholder="Enter your email address">
+        <button type="button">GET STARTED</button>
+      </div>
+
+      <!-- DESCRIPTIONS -->
+
+      <div class="description-container">
+        <ul>
+          <li class="list-description-container">
+            <img src="img/00.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Premium Materials</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+          <li class="list-description-container">
+            <img src="img/02.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Fast Shipping</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+          <li class="list-description-container">
+            <img src="img/01.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Quality Assurance</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <!-- YouTube Link -->
+
+      <div class="youtube-link">
+        <iframe width="1280" height="504" src="https://www.youtube.com/embed/65L_VV_Yepk" title="Zoe Wees - Girls Like Us (Piano Version)" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      </div>
+
```

Comment:
- i think its better if you can check your youtube embedded code and look for a way to fix it. 
- I hope if you can also fix the dimensions of the video on your page
![image](https://user-images.githubusercontent.com/59427923/198933476-8255403b-2781-4595-943d-5980d7b2446d.png)